File: C#/RefactoringWorkshop/RefactoringWorkshop/Coupler/FeatureEnvy/Employee.cs
namespace RefactoringWorkshop.Coupler.FeatureEnvy
{
    public class Employee
    {
        public int Age { get; set; }
        public int Grade { get; set; }
        public string Name { get; set; }

        public Employee(int age, int grade, string name)
        {
            Age = age;
            Grade = grade;
            Name = name;
        }
    }
}

File: Java/RefactoringWorkshop/src/main/java/coupler/featureenvy/Employee.java
package coupler.featureenvy;

public class Employee {
    private int age;
    private int grade;
    private String name;

    public Employee(int age, int grade, String name) {
        this.age = age;
        this.grade = grade;
        this.name = name;
    }

    public int age() {
        return age;
    }

    public int grade() {
        return grade;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setGrade(int grade) {
        this.grade = grade;
    }
}
